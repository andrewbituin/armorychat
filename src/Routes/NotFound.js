import React from 'react';
import './notFound.css'
import config from '../config';

export default function PageNotFound(){
  return ( 
    <div className='notFound-container'>
      <h1>404 Not Found</h1>
        <h3>What page were you looking for?</h3>
        <img className='NotFound' src={`${config.IMAGES_ENDPOINT}/whichone.gif`} alt='Jared from Silicon Valley'/>
    </div>
  )
}