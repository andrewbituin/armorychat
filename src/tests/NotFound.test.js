import React from 'react';
import NotFound from '../Routes/NotFound';
import ReactDOM from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

describe('NotFound component', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>, div
    );
    ReactDOM.unmountComponentAtNode(div);
  })

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <NotFound />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})