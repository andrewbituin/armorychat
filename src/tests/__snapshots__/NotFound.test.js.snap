// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders the UI as expected 1`] = `
<div
  className="notFound-container"
>
  <h1>
    404 Not Found
  </h1>
  <h3>
    What page were you looking for?
  </h3>
  <img
    alt="Jared from Silicon Valley"
    className="NotFound"
    src="https://limitless-brushlands-45977.herokuapp.com/images/whichone.gif"
  />
</div>
`;
